skbio/sequence/_sequence.py

```diff
@@ -21,6 +21,7 @@
 from contextlib import contextmanager
 
 import numpy as np
+from numpy.lib.stride_tricks import as_strided
```

Comment:
I would leave the full path from np. This is a relatively messy optimization and probably deserves the nasty fully qualified name